oap-server/server-storage-plugin/storage-iotdb-plugin/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>server-storage-plugin</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>8.9.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>storage-iotdb-plugin</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>server-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>library-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>iotdb-session</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.jpountz.lz4</groupId>
+                    <artifactId>lz4</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.lz4</groupId>
+            <artifactId>lz4-java</artifactId>
+            <version>1.6.0</version>
+        </dependency>
```

Comment:
I don't see there is `lz4` in `iotdb-session`'s dependency tree, please recheck

Reply:
Check the previous exclusive, the tricky thing here is, this repo changed the artifact ID. This is manual override dependency, and keep consistent with our existing dependency.

But the version should be managed in the bom.

Reply:
I mean there is no `lz4`, even `net.jpountz.lz4:lz4`, in `iotdb-session`

Reply:
OK, @LIU-WEI-git please recheck, I used to notice this jar in previous dependency check report.

Reply:
![image](https://user-images.githubusercontent.com/51618159/142755848-8b191ce2-d660-4cb1-8b8b-68832d623b19.png)
It still has lz4-1.3.0.

Reply:
@kezhenxu94 I can see this now.

![image](https://user-images.githubusercontent.com/5441976/142755865-37156e79-6a19-4275-9941-bba5128a08c2.png)

Reply:
OK, sorry @LIU-WEI-git , it does indeed include an old version of `lz4`, please add the `exclusion` back and move the new `lz4-java` to the `bom` module

Reply:
ok